features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesView.kt

```diff
@@ -188,6 +190,7 @@ fun MessagesView(
                     roomAvatar = state.roomAvatar.dataOrNull(),
                     heroes = state.heroes,
                     roomCallState = state.roomCallState,
+                    isDmUserVerified = state.dmUserVerificationState == IdentityState.Verified,
```

Comment:
I am not confortable with this comparison because `isDmUserVerified` will be false when the room is not a DM. If later we decide to render something else when this is a DM and the other user is not verified, we will have some trouble.

Since `dmUserVerificationState` is nullable, can you change this to 

```suggestion
                    isDmUserVerified = state.dmUserVerificationState?.let { it == IdentityState.Verified },
```

`isDmUserVerified` will need to be nullable as well.

Reply:
Ok, I mean if it's not a DM for sure the DM user is not verified (since there isn't one 😅 ) but maybe with the null value it'll be clearer.